.changeset/five-planets-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kdcio/api-gw-resp': patch
+---
+
+Improve documentation
```

Comment:
Improve documentation
May I also know what is the intention for this text?

Reply:
Para sa changeset yan